src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx

```diff
@@ -555,3 +469,83 @@ describe('SelectMultiple', () => {
     expect(screen.queryByRole('option', { name: /c selected/ })).not.toBeNull();
   });
 });
+
+describe('SelectMultiple with portal', () => {
+  window.scrollTo = jest.fn();
+  beforeEach(createPortal);
```

Comment:
Can you remind me what this is doing?

Reply:
This allows us to take snapshots of the drop